package.json

```diff
@@ -7,18 +7,21 @@
     "test": "stylelint test/*.css && stylelint test/*.scss --config sass.js"
   },
   "dependencies": {
-    "stylelint-config-standard": "^20.0.0",
-    "stylelint-order": "^4.0.0"
+    "stylelint-config-standard": "^24.0.0",
+    "stylelint-order": "^5.0.0"
   },
   "peerDependencies": {
-    "stylelint": "^13.3.2"
+    "stylelint": "^14.2.0"
   },
   "optionalDependencies": {
-    "stylelint-scss": "^3.16.1"
+    "stylelint-scss": "^4.1.0",
+    "postcss-scss": "^4.0.2"
   },
   "devDependencies": {
-    "stylelint": "^13.3.2",
-    "stylelint-scss": "^3.16.1"
+    "postcss": "^8.4.5",
```

Comment:
without postcss installed alongside postcss-scss, you will get the error `TypeError: Class extends value undefined is not a constructor or null at Object`

Reply:
I think the right way to handle this is to declare `postcss` as a peer dependency, and then add a `peerDependenciesMeta` block:

```json5
{
    "peerDependencies": {
        // ...
        "postcss": "^8.4.5"
    },
    "peerDependenciesMeta": {
        "postcss": {
            "optional": true
        }
    },
    "devDependencies": {
        // ...
        "postcss": "^8.4.5"
    }
}
```

See: 
* https://stackoverflow.com/questions/62047806/optional-peer-dependency 
* https://pnpm.io/package_json#peerdependenciesmetaoptional

Reply:
oh that's need, will update

Reply:
updated above